subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/GroovyGradlePluginInitIntegrationTest.groovy

```diff
@@ -52,6 +66,7 @@ class GroovyGradlePluginInitIntegrationTest extends AbstractInitIntegrationSpec
 
         where:
         scriptDsl << ScriptDslFixture.SCRIPT_DSLS
+        incubating << [true, false]
```

Comment:
This is a Spock gotcha. This isn't doing what you think it is.

Spock pulls one value from each list for each iteration, so this only runs twice. Once with Groovy and incubating=true and once with Kotlin with incubating=false.

To make this do both, you can do something like this:
```
[scriptDsl, incubating] << [
	ScriptDslFixture.SCRIPT_DSLS,
    [true, false]
].combinations()
```

Reply:
I was wondering why I saw that in a few places, it looked more complex than using 2 variables to me.